docs/source/conf.py

```diff
@@ -67,11 +67,12 @@
     'sphinx.ext.ifconfig',
     'sphinx.ext.mathjax',
     'sphinx.ext.viewcode',
-    'sphinx.ext.napoleon',
+    'numpydoc',
     'IPython.sphinxext.ipython_directive',
     'IPython.sphinxext.ipython_console_highlighting',
     'breathe',
-    'sphinx_tabs.tabs'
+    'sphinx_tabs.tabs',
+    'sphinx.ext.intersphinx'
```

Comment:
Hmm, can we order this list alphabetically?

Reply:
:+1: done